projects/adf5611/src/platform/platform_includes.h

```diff
@@ -0,0 +1,48 @@
+/***************************************************************************//**
```

Comment:
delete this file and change all occurences of `#include "platform_includes.h"` to `#include "parameters.h"`